time_tracker_api/projects/projects_model.py

```diff
@@ -124,6 +122,11 @@ def get_all(self, conditions: dict = None, **kwargs) -> list:
         add_customer_name_to_projects(projects, customers)
         return projects
 
+    def get_all_with_id_in_list(self,id_list):
```

Comment:
Maybe it should be discussed if the function will receive only one list because the find_all method supports two lists for sql in conditions.